### PR TITLE
Add rse-check

### PR DESCRIPTION
**Pull request by david-gimelle**

### CODE REVIEW

File: k8s/aat/common/rse/check.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name:rse-check
```
Comment (adusumillipraveen) [inline k8s/aat/common/rse/check.yaml:5]:
```suggestion
  name: rse-check
```

File: k8s/aat/common/rse/check.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    name: rse-check
+    version: 0.0.7
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rse/check:latest
+      ingressHost: rse-check-aat.service.core-compute-aat.internal
+      environment:
+        RESTART_ME: "Update due to vault change"
```
Comment (adusumillipraveen) [inline k8s/aat/common/rse/check.yaml:26]:
```suggestion
```
Shouldn't need this :)

File: k8s/prod/common/rse/check.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (adusumillipraveen) [inline k8s/prod/common/rse/check.yaml:1]: You cannot deploy to prod without an ATO. I know this is just a test, not sure though.

File: k8s/prod/common/rse/check.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rse-check
+    version: 0.0.7
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rse/check:latest
+      ingressHost: rse-check-prod.service.core-compute-prod.internal
```
Comment (adusumillipraveen) [inline k8s/prod/common/rse/check.yaml:24]: Ingress can move to base chart as templatedgenerally

File: k8s/prod/common/rse/check.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      environment:
+        RESTART_ME: "Update due to vault change"
+    global:
+      prometheus:
```
Comment (adusumillipraveen) [inline k8s/prod/common/rse/check.yaml:28]: Shouldn't be needed.

File: k8s/prod/common/rse/check.yaml (hunk 3 above)
Comment (david-gimelle) [reply k8s/prod/common/rse/check.yaml:1]: Yes. I don't think I really need to go to prod. I will remove this bit.

File: k8s/aat/common/rse/check.yaml (hunk 1 above)
Comment (david-gimelle) [reply k8s/aat/common/rse/check.yaml]: done

File: k8s/aat/common/rse/check.yaml (hunk 2 above)
Comment (david-gimelle) [reply k8s/aat/common/rse/check.yaml]: removed

File: k8s/prod/common/rse/check.yaml (hunk 4 above)
Comment (david-gimelle) [reply k8s/prod/common/rse/check.yaml]: I have remove the prod config

File: k8s/prod/common/rse/check.yaml (hunk 5 above)
Comment (david-gimelle) [reply k8s/prod/common/rse/check.yaml]: I have removed the prod config

**Pull request merged**